.github/workflows/run-prerelease.yml

```diff
@@ -0,0 +1,72 @@
+name: Release Activiti Cloud
+
```

Comment:
I think permissions should be configured for this workflow

Reply:
Since it's not a reusable workflow, it might be overkill, I'm keeping it open in case you'd like to refine this still, please disregard otherwise

Reply:
Let's handle permissions in the next iteration. I carried out some tests, and it seems the permissions only apply when the default GITHUB_TOKEN is used. It's not working with custom PATs.